### PR TITLE
FunctionApp with dotnet isolated runtime needs CLI 2.34.0 or lower

### PR DESCRIPTION
**Pull request by navba-MSFT**
FunctionApp with dotnet isolated runtime needs AzCLI 2.34.0 or lower.

Fixes Azure/azure-cli#22281

### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
  * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ You can place a breakpoint in any function, and inspect your code as it is runni
 
 ### Create the Azure resources
 
-1. To deploy the app, first ensure that you've installed the Azure CLI. 
+1. To deploy the app, first ensure that you've installed the Azure CLI. You need to use Azure CLI 2.3.4 version or below to create function app on dotnet isolated runtime.
```
Comment (satvu) [inline README.md:86]:
```suggestion
1. To deploy the app, first ensure that you've installed the Azure CLI. You need to use Azure CLI 2.3.4 version or below to create a .NET 5 function app on dotnet isolated runtime. Support for other .NET versions on Azure CLI are being tracked by [this issue](https://github.com/Azure/azure-functions-dotnet-worker/issues/857).
```

**Comment by fabiocav** [comment]
How long to we expect this to be the case? Are we fixing the feed issue to address the problem with the feed to address the issue?

**Comment by navba-MSFT** [comment]
@satvu committed the changes. Please review and approve this once you get a chance.

**Comment by satvu** [comment]
> How long to we expect this to be the case? Are we fixing the feed issue to address the problem with the feed to address the issue?

@fabiocav there's an issue tracking it [here](https://github.com/Azure/azure-functions-dotnet-worker/issues/857) which @mattchenderson created. I don't know when it will be fixed, but it seems the Stacks API fix is blocked on some other host issue.

**Comment by navba-MSFT** [comment]
@satvu Thanks for approving. Could you merge this once you get a chance ? Thanks in advance.

**Comment by fabiocav** [comment]
@navba-MSFT we're discussing whether we truly want to make a change to our guidance as opposed to working with the CLI team to have the bug fixed. We'll provide an update her soon.

**Comment by satvu** [comment]
@navba-MSFT We are going to close this as we are actively working on fixing the issue and do not believe it should be documented on the readme b/c it seems to suggest to users to downgrade azcli. We do not want to advise customers to downgrade in case they expose themselves to security vulnerabilities.

**Comment by runtimeware** [comment]
@satvu:  Can you link to the PR/issue where this problem is being worked?

**Pull request closed without merging**